From 1be0f7c365366925286ec1c2d2b88400f6c60518 Mon Sep 17 00:00:00 2001
From: Mikolaj Staworzynski <mikolaj.staworzynski@redembedded.com>
Date: Tue, 12 Sep 2023 17:13:12 +0200
Subject: [PATCH] ONEM-32192: switch libWPEFrameworkProcessContainers to use
 messaging

startup error:
wpeframework.sh[3085]: /usr/bin/WPEFramework: symbol lookup error: /usr/lib/libWPEFrameworkProcessContainers.so.4: undefined symbol: _ZN12WPEFramework5Trace9TraceUnit8InstanceEv
---
 Source/processcontainers/Tracing.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Source/processcontainers/Tracing.h b/Source/processcontainers/Tracing.h
index 55a83d9..0676137 100644
--- a/Source/processcontainers/Tracing.h
+++ b/Source/processcontainers/Tracing.h
@@ -21,7 +21,11 @@
 
 #include "Module.h"
 
+#ifdef __CORE_MESSAGING__
+#include "../messaging/messaging.h"
+#else
 #include "../tracing/tracing.h"
+#endif
 
 namespace WPEFramework {
 namespace ProcessContainers {
-- 
2.25.1

